FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Tool/appsettings.json

```diff
@@ -1,14 +1,20 @@
 {
   "fhirServer": {
-    "serverUrl": "",
-    "version": "R4",
```

Comment:
What's the consideration of removing version of FHIR server?

Reply:
Removed by mistake, add it back.